docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+In order to create a valid Integration DID message, the following steps must be executed:
+* Create and serialize a `diff` JSON object that contains all the changes. 
+* Set the `did` field to the DID value that this update applies to.
+* Retrieve the IOTA MessageId from the previous Diff chain message, or Integration DID message if this message is the first in the Diff chain. and a keypair for signing the DID Document.
+* Set the `previous_message_id` field to the IOTA MessageId value.
+* Create a `proof` object referencing the public key used inside the `verificationMethod` field and the signature suite in the `type` field. 
+* Create a cryptographic signature using the same keypair and add the result in the appropriate field(s) inside the `proof` object.
+
+### Delete
+
+In order to deactivate a DID document, an Integration DID message must be published that removes all content from a DID Document, effectively deactivating the DID Document. Keep in mind that this is irreversable. 
+
+## IOTA Identity standards
+
+The `did:iota` method is implemented in the IOTA Identity framework. This framework supports a number of operations that are standardized, some are standardized across the SSI community, and some are the invention of the IOTA Foundation. 
+
+### Standardized Verification Method Types
+
+We support two different Verification Method Types. Verification methods that can be used for signing DID Document, and verification methods that are only used for signing [W3C standardized Verifiable Credentials](https://www.w3.org/TR/vc-data-model/). This set differs as the IOTA Identity implements revocation through public key removal, which means every Verifiable Credential must be signed by a different keypair. We create collection of keypairs in a Merkle Tree in order to save space in the DID Document.
+
+Verification Methods that can be used to sign DID Documents and do other reusable activities:
+* `Ed25519VerificationKey2018` to create a `JcsEd25519Signature2020`.
+
+Verification Methods that can be used to sign Verifiable Credentials:
+* `MerkleKeyCollection2021` to create a `MerkleKeySignature2021`.
+
+### Revocation
+
+As mentioned above, revocation is done through deactivating public keys in the IOTA Identity framework. IOTA Identity has a highlky scalable solution since some identities might sign and want to revoke thousands if not millions of verifiable credentials. It is also GDPR compliant, leaving no trace, not even a hash, of a verifiable credential on the Tangle. Below will be a brief overview of how this mechanism works, but you may read all the details in the MerkleKeyCollection standardization document. 
```

Comment:
> through deactivating public

Through a revoke list... that's not really "deactivating" though, is it?

highlky -> highly
verifiable credential -> Verifiable Credential  (I think this is a name by now, no?)

MerkleKeyCollection standardization document link?

Reply:
It is according to the MerkleKeyCollection specification